akka-actor-tests/src/test/java/akka/event/LoggingAdapterTest.java

```diff
@@ -30,67 +33,72 @@
             "akka.actor.serialize-messages = off"
     );
 
-    @Test
-    public void mustFormatMessage() {
-        final ActorSystem system = ActorSystem.create("test-system", config);
-        try {
-            final LoggingAdapter log = Logging.getLogger(system, this);
-            new LogJavaTestKit(system) {{
-                system.eventStream().subscribe(getRef(), LogEvent.class);
-                log.error("One arg message: {}", "the arg");
-                expectLog(ErrorLevel(), "One arg message: the arg");
-    
-                Throwable cause = new IllegalStateException("This state is illegal") {
-                    @Override
-                    public synchronized Throwable fillInStackTrace() {
-                        return this; // no stack trace
-                    }
-                };
-                log.error(cause, "Two args message: {}, {}", "an arg", "another arg");
-                expectLog(ErrorLevel(), "Two args message: an arg, another arg", cause);
-    
-                int[] primitiveArgs = {10, 20, 30};
-                log.warning("Args as array of primitives: {}, {}, {}", primitiveArgs);
-                expectLog(WarningLevel(), "Args as array of primitives: 10, 20, 30");
-    
-                Date now = new Date();
-                UUID uuid = UUID.randomUUID();
-                Object[] objArgs = {"A String", now, uuid};
-                log.info("Args as array of objects: {}, {}, {}", objArgs);
-                expectLog(InfoLevel(), "Args as array of objects: A String, " + now.toString() + ", " + uuid.toString());
-    
-                log.debug("Four args message: {}, {}, {}, {}", 1, 2, 3, 4);
-                expectLog(DebugLevel(), "Four args message: 1, 2, 3, 4");
-    
-            }};
-        } finally {
-            JavaTestKit.shutdownActorSystem(system);
-        }
+    @Rule
+    public AkkaJUnitActorSystemResource actorSystemResource = new AkkaJUnitActorSystemResource("LoggingAdapterTest");
+
+    private ActorSystem system = null;
+
+    @Before
+    public void beforeEach() {
+    system = actorSystemResource.getSystem();
     }
 
     @Test
-    public void mustCallMdcForEveryLog() throws Exception {
-        final ActorSystem system = ActorSystem.create("test-system", config);
+    public void mustFormatMessage() {
+        final LoggingAdapter log = Logging.getLogger(system, this);
```

Comment:
would be good to settle on a java style. For streams / http 2-space is very important for it to not look completely crazy.
Shall we do that right away here too?

I was hoping to pull in @bantonsson's formatting sbt plugin soon